x-pack/legacy/plugins/maps/public/kibana_services.js

```diff
@@ -8,6 +8,7 @@ import { getRequestInspectorStats, getResponseInspectorStats } from 'ui/courier/
 export { xpackInfo } from 'plugins/xpack_main/services/xpack_info';
 import { start as data } from '../../../../../src/legacy/core_plugins/data/public/legacy';
 
+export const QUERY_DSL_FILTER_TYPE = data.filterTypes.QUERY_DSL;
```

Comment:
You can use anywhere inside the maps plugin by importing
`import { esFilters } from ''../../../../../src/plugins/data/public`